app/examples/github-actions/laravel-tests.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: [8.1, 8.2, 8.3]
+        php: ['8.1', '8.2', '8.3']
```

Comment:
What is the reason for this change? If it is that the generated BK pipeline needs them to be strings, we should have a bugfix on the translation to ensure they are always strings instead of changing them here as this would still be a valid GHA configuration we might find

Reply:
just a temporary fix for now to get the build green, ill work on ensuring its all strings soon

Reply:
I don't think is a good idea to change tests in order to have green builds, that could bring lots of problems, not to mention that it is bad practice. The bugfix doesn't seem to be very complicated 🤔

Reply:
hmm so a problem I'm having is that I can map the values to strings. But when its printed back to yaml, it seems to convert it back to numeric values again. I'm not sure why thats happening. See the latest commit for the mapping, but the output is still numeric

Reply:
left a comment with an explanation and way to solve the issue